assets/js/builder.index.entity.databasetable.js

```diff
@@ -159,8 +168,12 @@
         var $masterTabPane = this.getMasterTabsActivePane(),
             $form = $masterTabPane.find('form'),
             $toolbar = $masterTabPane.find('div[data-control=table] div.toolbar'),
-            $button = $('<a class="btn oc-icon-clock-o builder-custom-table-button" data-builder-command="databaseTable:cmdAddTimestamps"></a>')
+            $button = $('<a class="btn oc-icon-clock-o builder-custom-table-button" data-builder-command="databaseTable:cmdAddId"></a>')
```

Comment:
@DanHarrin The re-use of `$button` is a little bit of a code smell - would you be able to define this new button and the original "Add Timestamps" button with different variables? That way you can include them both in the `var` definition list.